$NetBSD: patch-src_archive_plugins_meson.build,v 1.2 2022/12/06 19:37:49 wiz Exp $

Fix building on MacOS when bzip2 and zziplib are disabled. The libarchive
plugin cannot be created since ar doesn't seem to want to create empty library.
Patch submitted upstream: https://github.com/MusicPlayerDaemon/MPD/issues/1650

--- src/archive/plugins/meson.build.orig	2022-11-28 15:55:46.000000000 +0000
+++ src/archive/plugins/meson.build
@@ -26,6 +26,7 @@ if not found_archive_plugin
   subdir_done()
 endif
 
+if libiso9660_dep.found() or libbz2_dep.found() or libzzip_dep.found()
 archive_plugins = static_library(
   'archive_plugins',
   archive_plugins_sources,
@@ -44,3 +45,4 @@ archive_plugins_dep = declare_dependency
     input_glue_dep,
   ],
 )
+endif
